src/aviary/message.py

```diff
@@ -124,16 +123,22 @@ def create_message(
         cls,
         role: str = DEFAULT_ROLE,
         text: str | None = None,
-        image: np.ndarray | None = None,
+        images: list[np.ndarray | str] | None = None,
```

Comment:
I'm not sure if this will break existing uses of `create_message` elsewhere. However, imo it's better to make this shift now while it isn't used much than in the future. It will be quite common for users to have multiple images associated with one prompt eg "what species do all the bacteria in these images belong to?". Single images can just be a list of length 1.

Reply:
cc. @maykcaldas for visibility